packages/contracts-bedrock/src/L1/DataAvailabilityChallenge.sol

```diff
@@ -154,7 +146,7 @@ contract DataAvailabilityChallenge is OwnableUpgradeable, ISemver {
         uint256 _bondSize,
         uint256 _resolverRefundPercentage
     )
-        public
```

Comment:
AFAICT visibility change has nothing to do with the PR right?

I wonder if there's a way we can encourage external over public when possible, tbh I thought slither was doing that for us. Not a blocker on this PR, but would be good to create an issue to look into it.

Reply:
If that's correct, I had copilot write one for me: 

Description:
The recent pull request #13183 included a visibility change from public to external for the initialize method. While this change was not directly related to the primary purpose of the PR, it highlights an opportunity to standardize our visibility preferences across the codebase.

Suggested Action:
Review the codebase to identify instances where public visibility can be replaced with external where appropriate.
Investigate why Slither is not flagging these instances and ensure it is configured to do so.
References:
Pull Request: [#13183](https://github.com/ethereum-optimism/optimism/pull/13183)
This standardization can help improve gas efficiency and align with best practices.

Reply:
oh lol, obviously this is enabled by removing the need to call the initializer in the constructor. Resolving.